code/modules/client/preferences.dm

```diff
@@ -251,118 +251,15 @@ datum/preferences
 /datum/preferences/proc/copy_to(mob/living/carbon/human/character, icon_updates = 1)
 	// Sanitizing rather than saving as someone might still be editing when copy_to occurs.
 	player_setup.sanitize_setup()

```

Comment:
Noting that you're making a sanitation call here, and then again for each preference datum in copy_to_mob.